src/app/components/error/index.html

```diff
@@ -279,7 +279,7 @@ <h3>Guidelines</h3>
       defaultValue="false"
       [isOptional]="true"
     >
-      Indicates whether to replace the default title. If <sky-code>false</sky-code>, this title is appended to the default title. This property accepts <sky-code>boolean</sky-code> values.
+      Indicates whether to replace the default title. If <sky-code>false</sky-code>, this title is appended to the default title. To replace the default title, the property must be wrapped in brackets. This property accepts <sky-code>boolean</sky-code> values.
```

Comment:
In the new sentence, I don't think we need the opening clause, but hopefully Alex can weigh in on what he thinks is the best way to communicate the this note about formatting in the code.

Reply:
@Blackbaud-AlexKingman Would you mind sharing your preference/thoughts on this?

Reply:
> To replace the default title, the property must be wrapped in brackets
I don't think this is necessary. Angular developers should know to use brackets for input properties. We don't call this out in other properties, so I don't think we'd need to do that here.